Examples/ArcGISToolkitExamples/Misc/ARStatusViewController.swift

```diff
@@ -97,6 +115,39 @@ class ARStatusViewController: UITableViewController {
         }
     }
 
+    /// The horizontal accuracy of the last location.
+    public var horizontalAccuracy: Double = 1.0 {
+        didSet {
+            guard horizontalAccuracyLabel != nil else { return }
```

Comment:
A couple things:
1. If this was set from a background queue, the label could become non-nil by the time the work would have been dispatched to the main queue.
2. You could just use optional chaining:

    ```swift
    self.horizontalAccuracyLabel?.text = String(format: "%.0f", self.horizontalAccuracy)
    ```

Same with the other new properties.

Reply:
There's no reason this same pattern couldn't be applied to all the properties, so that's what I did.  Thanks!